examples/cordova-sample/MyApp/bin/refresh_javascript_sources_ios.sh

```diff
@@ -1,3 +1,17 @@
+#!/bin/bash
+# Environment detection
```

Comment:
When running a script from Xcode we need to load up homebrew stuff so we can access coreutils
This ensures we're compatible with both apple and intel chips.